spec/lib/rbac/filterer_spec.rb

```diff
@@ -2540,6 +2552,15 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
 
       results = described_class.search(:class => Flavor, :user => other_user).first
       expect(results).to match_array [project1_flavor, project2_flavor, flavor_other]
+
+      results = described_class.search(:class => OrchestrationStack, :user => project1_user).first
+      expect(results).to match_array [project1_orchestration_stack, project2_orchestration_stack, orchestration_stack_other]
```

Comment:
if we are querying all OrchestrationStacks so here can be
``OrchestrationStack.all`` instead of `[project1_orchestration_stack, project2_orchestration_stack, orchestration_stack_other]`

I guess it is more visible